### DISCUSSION TITLE
Library show addon like itunes

### DISCUSSION THREAD

**Discussion by giorgiobrit** [TT-ReBORN/Georgia-ReBORN#146] [Ideas]
Hi Tom, it would be great to have the ability to be able to click 2 times on the album cover and see below the list of songs that includes the album. Like iTunes interface. Would something like that be possible?
![screenshot 2](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/57190501/7cfcc7c5-1f05-4998-b3ad-d6508f43971e)
<img width="1280" alt="Screen-Shot-2012-11-29-at-2 25 25-PM" src="https://github.com/TT-ReBORN/Georgia-ReBORN/assets/57190501/575e8273-beab-41ef-ace6-b4f6a6ed2977">

**Comment by TT-ReBORN** [comment]
This was already requested by some users in the past:
https://hydrogenaud.io/index.php/topic,121047.msg1025277.html#msg1025277
https://hydrogenaud.io/index.php/topic,121047.msg1031429.html#msg1031429
It will be implemented in the future, it is not an easy task...

**Reply by giorgiobrit** [reply]
No worries but can't wait ! Thanks

**Comment by Tong-Kam** [comment]
Hope this feature can be implemented as soon as possible, Georgia-ReBORN is the best!！！！

**Reply by fwn0** [reply]
Yes, this new feature will be available very soon. It is a huge step forward in the evolution of Georgia-Reborn, possibly called 4.0. This is not just a cheap clone of the iTunes interface, but a much more complex extension with new ideas that have been implemented in Georgia-Reborn. The code is finished, just needs more testing and polishing before it can be released soon. I am looking forward to testing this new version tonight on Linux (and maybe Windows too). :-)
